### ISSUE TITLE
Missing fields for STB EMU / Stalker Portal

### ISSUE THREAD

**Issue by GTP000T** [4gray/iptvnator#345]
Hello and a happy new year.

Necessary fields are missing for STB EMU / Stalker Portal's. Add fields for serial number, device id 1, device id2, signature. Good to have is user agent and request method Get/Post. 

Thank you, good work so far :)

**Comment by thenoobtester** [comment]
I agree, my STB Stalker portals work but not with this software, which is a great software for m3u8 lists and xtream codes nonetheless.

+1
